.github/scripts/cleanup-cluster.sh

```diff
@@ -22,13 +22,13 @@ echo "cleaning up cluster"
 
 # Delete all test resources in queuemessages.
 echo "delete all resources in queuemessages.ucp.dev"
-kubectl delete queuemessages.ucp.dev -n radius-system --all
+kubectl delete queuemessages.ucp.dev -n radius-system --all || true
```

Comment:
Is there any potential downside in ignoring failures here?

Reply:
This is not the final code. I am still working on it.

Reply:
Ok, I'll stop looking at it until it is ready then.

Reply:
please review it again.